app.js

```diff
@@ -18,40 +19,55 @@ go();
 async function go() {
   try {
     for (const org of config.orgs) {
+      console.log('⏳ Collecting repos for', org)
       let page = 1;
       while (true) {
         const params = {
           page,
-          access_token: config.token
+          sort
         };
         const url = `https://api.github.com/orgs/${org}/repos?${qs.stringify(params)}`;
+
         const repos = await request(url, {
           json: true,
           headers: {
-            'User-Agent': 'changelog-scanner'
+            'User-Agent': 'changelog-scanner',
+            'Authorization': `token ${config.token}`
           },
         });
+
+        if (!Array.isArray(repos)) {
+          console.error('Error retrieving repos:', repos);
+          return;
+        }
+
         allRepos = allRepos.concat(repos.map(repo => `${org}/${repo.name}`));
         if (!repos.length) {
           break;
         }
+        console.log(`⏳ ${repos.length} repos found. Checking for more...`)
         page++;
       }
+
+      console.log(`⏳ Found ${allRepos.length} repos for ${org}`)
     }
+
     for (const repo of allRepos) {
+      console.log(`⏳ Checking commits for ${repo}`)
       let page = 1;
       let allCommits = [];
       while (true) {
         const params = {
           page,
           since,
-          access_token: config.token
+          'Authorization': `token ${config.token}`
```

Comment:
Does this make sense in params where it becomes part of the query string still? I see you also have it in the request call as a header.

Reply:
No, I think that was a mistake.